server/app/routes/kommissions.js

```diff
@@ -0,0 +1,14 @@
+const router = require('express').Router();
+const guard = require('express-jwt-permissions')();
```

Comment:
Unused variable, *eslint* will report this anyway
`guard` check will be done later